arm:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  joint_group_controller:
    type: position_controllers/JointGroupPositionController
    joints: 
     - rotation_joint
     - lower_joint
     - mid_joint
     - upper_joint
     - wrist1_joint
     - wrist2_joint
     - wrist3_joint

  gripper_controller:
    type: position_controllers/GripperActionController
    joint: robotiq_85_left_knuckle_joint
    action_monitor_rate: 20
    goal_tolerance: 0.002
    max_effort: 100
    stall_velocity_threshold: 0.001
    stall_timeout: 1.0
     